src/components/ItaliaTheme/View/DocumentoView/DocumentoDescrizione.jsx

```diff
@@ -57,10 +57,7 @@ const DocumentoDescrizione = ({ content }) => {
         />
       )}
       {content.autori?.length > 0 && (
-        <CuredBy
-          people={content.autori}
-          title={intl.formatMessage(messages.autori)}
-        />
+        <DocumentoAutori autori={content.autori} />
```

Comment:
Il componente `CuredBy` non viene più usato togliendolo da qui? In tal caso metterei un commento in cima al file di quel componente che dice che non viene più usato, oppure ragioniamo direttamente di toglierlo. Parliamone, in caso non venga più usato.